fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -449,4 +449,9 @@ private String txSegment() {
             return EMPTY;
         }
     }
+
+    @Override
```

Comment:
Not supporting `hashCode()` seems like an a sign of a more fundamental issue.
But any ways, the related Sonar issue seems to be gone: http://sonar.fcrepo.org/drilldown/issues/1?&rule=findbugs%3AEQ_COMPARETO_USE_OBJECT_EQUALS&rule_sev=MAJOR&severity=MAJOR